src/graphics/device.js

```diff
@@ -879,21 +879,22 @@ Object.assign(pc, function () {
             this.transformFeedbackBuffer = null;
         },
 
-        initializeGrabPassTexture: function () {
+        initializeGrabPass: function (options) {
             if (this.grabPassTexture) return;
 
+            var format = options.alpha ? pc.PIXELFORMAT_R8_G8_B8_A8 : pc.PIXELFORMAT_R8_G8_B8;
+
             var grabPassTexture = new pc.Texture(this, {
-                format: pc.PIXELFORMAT_R8_G8_B8_A8,
-                autoMipmap: false
+                format: format,
+                minFilter: pc.FILTER_LINEAR,
+                magFilter: pc.FILTER_LINEAR,
+                addressU: pc.ADDRESS_CLAMP_TO_EDGE,
+                addressV: pc.ADDRESS_CLAMP_TO_EDGE,
+                mipmaps: false
             });
 
-            grabPassTexture.minFilter = pc.FILTER_LINEAR;
-            grabPassTexture.magFilter = pc.FILTER_LINEAR;
-            grabPassTexture.addressU = pc.ADDRESS_CLAMP_TO_EDGE;
-            grabPassTexture.addressV = pc.ADDRESS_CLAMP_TO_EDGE;
-
+            grabPassTexture._pot = false; // never treat texture as "power of 2", framebuffer size is arbitrary.
```

Comment:
I'm not super-keen on _pot being accessed here, TBH. Looking at pc.Texture, we could probably lose this property and just calculate on demand since it's not really performance critical. But let's leave that for another PR.

Reply:
I can just set `width` and `height` to zero.